peep.py

```diff
@@ -104,6 +104,11 @@ def iter(self, ret, *args, **kwargs):
 
     DownloadProgressBar = DownloadProgressSpinner = NullProgressBar
 
+try:
+    from pip.index import FormatControl  # 7.0
```

Comment:
Change the comment here to ``  # noqa`` to silence the flake8 unused import warning. (And perhaps move the 7.0 comment to the next line).